### PR TITLE
Create arielTROY.md

### PR DESCRIPTION
**Pull request by arielTROY**
## Describe your project in 1-3 sentences

a smart glasses that helping blind with ai
## Hack Club Slack handle
@arielT

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by arielTROY** [comment]
@bellesea Sorry for the tagging, but can you review my project? It's been two days and I need to order the parts quickly so they arrive on time

File: arielTROY.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+but maybe i will test tensorflow for micro devices without phone and compare the latency and results between the two methods to see which better
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| bluetooth headphones|at home   | $0 |\
+| wires | at home  | $0 |
+| esp32cam board |https://www.amazon.com/Aideepen-ESP32-CAM-Bluetooth-ESP32-CAM-MB-Arduino/| $12.99 | free shipping
```
Comment (bellesea) [inline arielTROY.md:34]: hiii :) can you please update this link? it leads to a page not found for me... is it this link? https://www.amazon.com/Aideepen-ESP32-CAM-Bluetooth-ESP32-CAM-MB-Arduino/dp/B08P2578LV

File: arielTROY.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: "@arielTROY"
+project: "forblind"
+---
+
+# Project Name
```
Comment (bellesea) [inline arielTROY.md:6]:
```suggestion
# forblind
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hey @arielTROY this is super cool!

a few questions:
- could you please expand on how this would work technically? how do you know this 
- would you simultaneously run the ocr model, the image classification model, and the voice reognition model on your board? if so, does the esp32 cam board have the capabilties to do it all real time?
- would you be designing the glasses yourself? if they are sunglasses do you need special film or smth?
- do you need a tf card?

please commit the relevant changes in your file :) thank you!

File: arielTROY.md (hunk 1 above)
Comment (arielTROY) [reply arielTROY.md]: updated

**Comment by arielTROY** [comment]
@bellesea thank you for checking my project! :) can you please check again the changes now? Sorry to be in a hurry but Amazon simply says that the products will probably arrive very close to the start of the competition if at all.
Thank you for understanding

**Review by bellesea — APPROVED** [review]
@arielTROY thanks for expanding on your plan!

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps :)

approving to move forward but instead of the "updating" section could you please integrate your responses into your plan? also, will you be 3d printing the case yourself?

**Pull request merged**